bundle/android/strings.xml

```diff
@@ -320,6 +319,7 @@ translators: %s: Select control option value e.g: "Auto, 25%". -->
     <string name="gutenberg_native_take_a_video" tools:ignore="UnusedResources">Take a Video</string>
     <string name="gutenberg_native_tap_here_to_show_help" tools:ignore="UnusedResources">Tap here to show help</string>
     <string name="gutenberg_native_tap_to_hide_the_keyboard" tools:ignore="UnusedResources">Tap to hide the keyboard</string>
+    <string name="gutenberg_native_text_color" tools:ignore="UnusedResources">Text color</string>
```

Comment:
This is a new string that has been recently added ([PR reference](https://github.com/WordPress/gutenberg/pull/36028)).

- Referenced in `gutenberg/packages/format-library/src/text-color/index.native.js` ([reference](https://github.com/WordPress/gutenberg/blob/b7d6b2371a85c550f0205be424377fa986eac3bd/packages/format-library/src/text-color/index.native.js#L27:L27
))
- Already translated in Gutenberg GlotPress project ([reference](https://translate.wordpress.org/projects/wp-plugins/gutenberg/dev/en-gb/default/?filters%5Bterm%5D=Text+color&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Bcase_sensitive%5D=yes&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)) but since it's only referenced in a `*.native.js` file, it will most likely be removed eventually.